decidim-verifications/README.md

```diff
@@ -75,6 +75,49 @@ Decidim implements two type of authorization methods:
   * `edit_authorization_path`: This is the entry point to resume an existing
     authorization process.
 
+## Custom action authorizers
+
+Custom action authorizers are an advanced feature that can be used in both types of
+authorization methods to customize some parts of the authorization process.
+These are particulary useful when used within verification options, which are
+set in the admin zone related to a feature action. As a result, a verification
+method will be allowed to change the authorization logic and the appearance based
+on the context where the authorization is being performed.
```

Comment:
Maybe mention an example so a developer or a user can understand its value?

Reply:
And also an example on how to set it at the admin zone (like the screenshot)

Reply:
What do you think about [this](https://github.com/podemos-info/decidim/blob/1925943249afb2a05dac0907fb32b9d60b7ba6f9/decidim-verifications/README.md#custom-action-authorizers)? I'm not very good in english developer-human communications, any help would be very appreciated. 😅